sr_demos/scripts/sr_demos/demo.py

```diff
@@ -405,11 +437,13 @@ def standard_demo_sequence(self):
         self.execute_command_check('zero_wr', 0.4)
         self.commander.move_to_named_target("open")
         rospy.loginfo("Standard demo completed")
+        self._enable_touch_detection(True)
 
     def rock_paper_scissors(self):
```

Comment:
No type hinting?

Reply:
fixed in https://github.com/shadow-robot/sr_interface/pull/823/commits/4d899740e14c6a37b20146403ea06d3b82a837c8